src/transceivercontroller/DefaultTransceiverController.ts

```diff
@@ -217,20 +217,127 @@ export default class DefaultTransceiverController implements TransceiverControll
     videosToReceive: VideoStreamIdSet
   ): void {
     const videosRemaining = videosToReceive.array();
+    if (transceivers.length !== 0 && !transceivers[0].stop) {
```

Comment:
Can we add this to V3 for consideration? Seem like a straightforward low hanging fruit change :D

Reply:
Yup, maybe we can just do a grep for version numbers/webrtc function checks in the codebase since I'm sure there are other checks like this.  We can also raise limits based off usage.